packages/xo-web/src/xo-app/settings/logs/index.js

```diff
@@ -55,15 +73,7 @@ const COLUMNS = [
         <pre className={styles.widthLimit}>
           {log.data.error && log.data.error.message}
         </pre>
-        {log.data.error && log.data.error.code === 'LICENCE_RESTRICTION' && (
-          <a
-            href='https://xcp-ng.org/'
-            rel='noopener noreferrer'
-            target='_blank'
-          >
-            {_('logSuggestXcpNg')}
-          </a>
-        )}
+        <LogInfo log={log} />
```

Comment:
What do you think about:  `Log` component that displays ` error` and `info`?

Reply:
To replace the whole `itemRenderer` of this column?

Reply:
ping @Rajaa-BARHTAOUI